test/integration/scheduler/scheduler_test.go

```diff
@@ -626,7 +657,12 @@ var _ = ginkgo.Describe("Scheduler", func() {
 		})
 
 		ginkgo.AfterEach(func() {
+			gomega.Expect(framework.DeleteNamespace(ctx, k8sClient, ns)).To(gomega.Succeed())
 			gomega.Expect(framework.DeleteClusterQueue(ctx, k8sClient, cq)).ToNot(gomega.HaveOccurred())
+			gomega.Eventually(func() error {
+				var newCQ kueue.ClusterQueue
+				return k8sClient.Get(ctx, client.ObjectKeyFromObject(cq), &newCQ)
+			}, framework.Timeout, framework.Interval).Should(testing.BeNotFoundError())
```

Comment:
can we create a function named `WaitUntilForClusterQueueDeletion(cq)` and use it everywhere?